### DISCUSSION TITLE
PyCon India 2021 Keynote Speaker Suggestions

### DISCUSSION THREAD

**Discussion by rayandas** [pythonindia/inpycon2021-tasks#19] [General]
Please mention your keynote speaker suggestions here. It should contain a brief description about the speaker or include a link to the bio of the speaker. 
If you like a speaker suggestion, please vote by writing a line about your vote for e.g. "I would also love to hear X as a keynote speaker [because of these Y reasons]".

The selected keynote will be a mix of Indian and International speakers, so feel free to suggest from both.

We will start reaching out to keynote speakers in the order of preference within the next few weeks. Please suggest your keynote speakers by 19 March 20:00 IST.

**Comment by raukadah** [comment]
Abhilash Raj , He is GNU Mailman lead developer and CPython core developer.
Twitter: [https://twitter.com/_maxking_](https://twitter.com/_maxking_)
Github: https://github.com/maxking
Blog:  https://asynchronous.in/

**Comment by avidLearnerInProgress** [comment]
Raymond Hettinger => A phenomenal PyCon speaker over many years.
Twitter: https://twitter.com/raymondh

**Comment by indrag49** [comment]
Dr. Kathryn Huff. She is an assistant professor at the NPRE, University of Illinois, Urbana-Champaign. She is a brilliant speaker and has been delivering talks at Scipy conferences and Pycons.

Website: http://kdhuff.npre.illinois.edu/

**Comment by deb17** [comment]
Miguel Grinberg - he is very popular in the Python community for his books and tutorials on Flask.

Twitter  - https://twitter.com/miguelgrinberg
Blog - https://blog.miguelgrinberg.com

**Comment by eumiro** [comment]
**Luciano Ramalho**, author of the book _Fluent Python_ (2nd edition soon to be published) always finds the “beautiful” in the Python language.

Twitter: https://twitter.com/ramalhoorg

**Comment by animator** [comment]
**Lorena Barba**, member of the Board of Directors of NumFOCUS, member of the editorial board for IEEE/AIP Computing in Science and Engineering, The Journal of Open Source Software, and The ReScience Journal. She founded The Journal of Open Source Education, and is its Editor-in-Chief. She is also the recipient of the 2016 Leamer-Rosenthal Award for Open Social Sciences. She was nominated and received an Honorable Mention in the 2017 Open Education Awards for Excellence of the Open Education Consortium.

Twitter: https://twitter.com/LorenaABarba
About Me: https://about.me/lorenabarba

**Comment by saravanansaminathan** [comment]
Noordeen is a Senior Data scientist in Zoho corporation for more than 4 years. He worked on CNN, image analystics and find the application of Deep learning in CRM.
He is passionate about teaching and spread AI knowledge among college community students. He is Passionate about training people in Machine learning and Deep learning competencies who are inquisitive to learn AI and thereby contributing to the democratization of AI to solve real-world problems and to create the greatest improvements in quality of life the world has ever seen.

Linkedin: https://www.linkedin.com/in/mknoordeen/
Medium: https://medium.com/@mdnoorit
Github: https://github.com/mdnoorit
Twitter: https://twitter.com/mdnoorit

**Comment by salman0149** [comment]
David Beazley,
Raymond Hettinger

**Comment by navinpai** [comment]
**François Chollet** ([@fchollet on Twitter](https://twitter.com/fchollet) | [Website](https://fchollet.com/))
- Author and Maintainer of Keras, which is an open-source software library that provides a Python interface for artificial neural networks. Also the author of 'Deep Learning with Python'

**Comment by Pradhvan** [comment]
**Andrew Godwin** [Twitter](https://twitter.com/andrewgodwin) | [Website](https://aeracode.org/about/)

* Andrew is a Django core developer and has made some significant contributions to Django like migrations, Asynchronous support, and much more. 

**Carlton Gibson** [Twitter](https://twitter.com/carltongibson) | [Website](https://noumenal.es/)

* Carlton has been a long-time Django contributor and Django Fellow. Besides Django, he also helps maintain DRF, Django-Filter, Channels, Crispy Forms, and co-host a weekly podcast Django Chat. 


**Jessica McKellar** [Twitter](https://twitter.com/jessicamckellar) 

* Jessica has been a director of the Python Software Foundation and has made significant efforts to increase the diversity of open-source communities. 

**Thea Flowers**  [Twitter](https://twitter.com/theavalkyrie) | [Website](https://thea.codes/stuff.html)

* Thea is a Python Software Foundation Fellow and helps maintain urllib3, Twine, and packaging.python.org. She also volunteers as an organizer, speaker mentor, and diversity expert at several conferences like PyCascades and more.

**Reply by SaptakS** [reply]
+1 for Jessica McKellar

**Comment by IHackPy** [comment]
Joris Van den Bossche [Twitter](https://twitter.com/jorisvdbossche)

- Pandas Core developer. GeoPandas maintainer.

Marc Garcia [Website](https://datapythonista.me/)

- Pandas Core Developer

**Comment by bravegnu** [comment]
Anima Anandkumar

Quoting from https://research.nvidia.com/person/anima-anandkumar

Animashree (Anima) Anandkumar joined NVIDIA as director of machine learning research in 2018. She is also a Bren professor at Caltech CMS department since 2017. Her research spans both theoretical and practical aspects of  machine learning. In particular, she has spearheaded research in tensor-algebraic methods, large-scale learning, deep learning, probabilistic models, and non-convex optimization. 

Her keynote at PyData: https://www.youtube.com/watch?v=VL4NC4f6m0w

**Comment by bravegnu** [comment]
Prof. Anant Agarwal

Quoting from https://en.wikipedia.org/wiki/Anant_Agarwal

Anant Agarwal is an Indian computer architecture researcher.[1] He is a professor of Electrical Engineering and Computer Science at the Massachusetts Institute of Technology (MIT), where he led the development of Alewife, an early cache coherent multiprocessor, and also has served as director of the MIT Computer Science and Artificial Intelligence Laboratory. He is the founder and CTO of Tilera, a fabless semiconductor company focusing on scalable multicore embedded processor design.[2] He also serves as the CEO of edX, a joint partnership between MIT and Harvard University that offers free online learning.

Quoting from https://en.wikipedia.org/wiki/EdX

Open edX platform is the open-source platform software developed by edX and made freely available to other institutions of higher learning that want to make similar offerings. On June 1, 2013, edX open sourced its entire platform.[20] The source code can be found on GitHub.[21][22]

The Open edX server-side software is almost entirely based on "Python, with Django as the web application framework."[23] 

TED Talk: https://www.youtube.com/watch?v=rYwTA5RA9eU

**Comment by wavicles** [comment]
Jithin B P
A passionate young scientist, maker and entrepreneur. 
Jithin has been developing Python powered scientific instrumentation tool for highschool kids and also for high energy nuclear research.
He is involved in teacher training programs promoting use of python at various levels. Currently he is working on designing python powered tools for science demos in classroom and open source science hardware for school laboratories. Has presented his work at various national and international conferences including RMLL and Scipy. 

Following are some of the python based software and hardware frameworks designed by Jithin.

Kuttypy : https://csparkresearch.in/kuttypyl
VLabTool : http://jithinbp.in/a-versatile-labtool/
New Hardware for ExpEYES : https://csparkresearch.in/expeyes17/

Twitter: https://twitter.com/jithinbp_
Github: https://github.com/jithinbp
Github: https://github.com/csparkresearch
Website: http://jithinbp.in/

**Comment by wavicles** [comment]
Dr. AjithKumar B P

(Gave a keynote at PyCon India 2015, Kenneth Gonsalves Award 2016)
Retired as a senior scientist from  Inter University Accelerator Centre, New Delhi, in Feb-2021. 
He has done his M.Sc and PhD in Nuclear Physics from University of Calicut and M.Sc in Accelerator Physics, from University of Manitoba. Mainly involved in the design and development of particle accelerators and associated experimental facilities, that involves using a lot of open source tools. 
For the past several years he has been involved in promoting Free Software tools in Education and leading a project for developing open hardware laboratory equipment for teaching science, that uses Python for data analysis and visualization. Has been pursuing academic community to include Python in their syllabus and has written a book 'Python for Educatio'. Dr. Ajith is involved in teacher training programs and has helped thousands of teachers to take their first steps in the world of python.

Following are the links to his work in promoting python in education.

ExpEYES : https://expeyes.in/index.html
Python for Schools: https://expeyes.in/python.html
Project Phoenix : https://www.iuac.res.in/phoenix/

Twitter: https://twitter.com/AjithKumarBP
Website: https://scischool.in/

**Comment by niharikakrishnan** [comment]
[Dr. Fei Fei Li](https://en.wikipedia.org/wiki/Fei-Fei_Li)

Quoting from [https://profiles.stanford.edu/fei-fei-li](https://profiles.stanford.edu/fei-fei-li)
Dr. Fei-Fei Li is the inaugural Sequoia Professor in the Computer Science Department at Stanford University, and Co-Director of Stanford’s Human-Centered AI Institute. She served as the Director of Stanford’s AI Lab from 2013 to 2018. And during her sabbatical from Stanford from January 2017 to September 2018, she was Vice President at Google and served as Chief Scientist of AI/ML at Google Cloud.

Quoting from [https://anitab.org/profile/fei-fei-li/](https://anitab.org/profile/fei-fei-li/)
An unprecedented thought leader in AI through her revolutionary computer vision research, Fei-Fei has had transformational industry impact democratizing AI, pioneering future technological innovations, and advocating diversity in STEM and AI internationally. Fei-Fei is the lead inventor of ImageNet. It was by far the largest publicly available dataset at the time of its release in 2009, and its unprecedented depth and detail had a transformative effect on a class of algorithms known as Convolutional Neural Networks.

Talk at Grace Hopper Conference: [https://www.youtube.com/watch?v=XlnbNFW2tX8](https://www.youtube.com/watch?v=XlnbNFW2tX8)

Ted Talk: [https://www.youtube.com/watch?v=40riCqvRoMs](https://www.youtube.com/watch?v=40riCqvRoMs)

**Comment by SaptakS** [comment]
[Nina Zakharenko](https://www.nnja.io/)

Quoting from https://us.pycon.org/2019/keynote-speakers/
> Nina Zakharenko is a software engineer with over a decade of experience. She currently focuses on Python at Microsoft on the Cloud Developer Advocacy team. In the past, she’s written software for satellite control computers at HBO, code that's helped people connect over their passions at Meetup, and implemented time-wasting features on Reddit.
Nina loves teaching developers and has spoken at conferences like PyCon Russia, EuroPython, and DjangoCon in the US and internationally. This year, she's looking forward to attending her seventh PyCon, where you'll find her co-organizing Mentored Sprints for Diverse Beginners.

**Comment by SaptakS** [comment]
[Parisa Tabriz](https://asirap.net/)

Parisa Tabriz has worked on information security for over a decade and as a (self-appointed) “Security Princess” of Google. She started as a “hired hacker” software engineer for Google’s security team.

Paraphrased from her website:
> She has been focusing on making the web safer to browse so users don't get pwnd, making Internet traffic more secure, spreading the hacker mindset, and trying to improve diversity in the tech industry because it leads to more innovation and better results. She has also worked with the U.S. Digital Service to enhance network and software security at the White House. 

https://www.elle.com/culture/tech/a14652/google-parisa-tabriz-profile/

**Comment by pnijhara** [comment]
### Marlene
A Zimbabwean explorer, speaker and cucumber enthusiast based in the city of Harare. She is an advocate for using science and technology for social good and increasing diversity in these fields. In 2017, she co-founded ZimboPy, a non-profit organisation that gives Zimbabwean young women access to resources in the field of technology. She is a director and vice-chair for the Python Software Foundation, and the chair of PyCon Africa. She is also currently pursuing a computer science degree at the University of London

website: https://marlenemhangami.com/
Twitter: https://twitter.com/marlene_zw

**Comment by palnabarun** [comment]
Thank you everyone for your suggestions. 🎉 

We've locked the conversation since we're past the keynote suggestions collection phase.
